docs/zh/latest/FAQ.md

```diff
@@ -501,3 +501,4 @@ $ acme.sh --renew --domain demo.domain
 ```
 
 详细步骤，可以参考博客 https://juejin.cn/post/6965778290619449351
+
```

Comment:
Please remove the blank line which causes the failure: https://github.com/apache/apisix/pull/4362/checks?check_run_id=2751014883

Reply:
I am sorry.I want to know if there is a way to let me know my mistakes like this blank in my native environment before I commit.Thanks

Reply:
@ysc972 we haven't provided a one-line method, but you could follow the actions step (for this is using npm to install `markdownlint-cli`) to test locally

Reply:
@Yiyiyimu Thank you . I wii try it .